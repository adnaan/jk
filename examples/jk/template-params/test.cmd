jk run ./template.js -p selector.foo=bar -p name=foosvc
