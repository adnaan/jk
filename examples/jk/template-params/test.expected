apiVersion: v1
kind: Service
metadata:
  name: foosvc
  namespace: default
spec:
  ports:
  - port: 8080
  selector:
    foo: bar
